packages/cli/src/validate-package.ts

```diff
@@ -36,6 +40,8 @@ export async function fixPackage(pkg: Package) {
 
   pkg.json = setFieldInOrder(pkg.json, "exports", exportsField(pkg));
 
+  await pkg.save();
```

Comment:
hm, it's somewhat surprising that this was added here - is there any specific reason for it? why wasn't it here before? how did this feature make it required?

Reply:
Previously there were no fixes that happened on a package, only entrypoints but the exports field goes on a package, not an entrypoint